README.md

```diff
@@ -123,6 +156,55 @@ git clone https://github.com/peng-lab/PyBaSiC.git
 For mitosis movie frames, we give PyBaSiC 2 frames before/after the desired frame (depending on position in movie) to perform illumination correction.
 We discuss different methods of illumination correction in [#1](https://github.com/WayScience/IDR_stream/issues/1).
 
+### Fiji:
+
+Download [Fiji](https://imagej.net/software/fiji/downloads) using the appropriate file for your OS.
+We recommend putting the Fiji application file (for Linux it is called `Fiji.app`) that comes from the install on your Desktop or in a place that you can easily find the path to.
+
+### CellProfiler:
+
+CellProfiler will be used for both segmentation and feature extraction.
+For segmentation, we recommend using the Cellpose plugin for CellProfiler for projects centered around nuclei. 
+Cellpose and CellProfiler are already installed from source within the conda environment from the `Necessary Packages` section.
+
+To run the CellProfiler Cellpose plugin, the steps are as follows:
+
+#### Step 1: Clone CellProfiler Github
+
+We recommend putting it into a folder on your Desktop called `GitHub` for better organization and tracking of the repo.
+
+```console
+# Make sure to `cd` into the directory that you want the repo in (e.g. \~/Desktop/Github)
+git clone https://github.com/CellProfiler/CellProfiler.git
```

Comment:
Does this mean CellProfiler gets installed into a different folder than `IDR_stream`? If so this might be bad...

For example, when we have to use PyBaSiC and DeepProfiler we install those **within** the cloned `IDR_stream/` folder.
I am pretty sure if you clone `IDR_stream/` into `/Desktop/Github` then cd into `/Desktop/Github` and use `git clone` this will put `CellProfiler` into `/Desktop/Github`, not into `/Desktop/Github/IDR_stream`.
Maybe you meant to say the user should cd into `/Desktop/Github/IDR_stream`?

Reply:
I am reading a bit further into the README now and think I might have a better idea of how this setup works.

Is the user setting up CP for general use (and installing into `/Desktop/Github/`) and then we utilize the general use CP installation for `IDR_stream`?
Even so, I get a little worried about putting stuff related to `IDR_stream` outside of `/Desktop/Github/IDR_stream` but you can judge if that needs to change.

Reply:
Yes to the second part. But it isn't really setup, its mainly just putting the repo somewhere and then being able to put the plugin into the repo for Cellprofiler to reference. We can discuss this on Friday regarding where it is best to put it. It is kinda like for Fiji where the user installs it on their computer not into IDR_stream directly.

Reply:
Ahh that's a good point. Let's discuss Friday.